### PR TITLE
added asadmin command stop-all-domains

### PR DESCRIPTION
**Pull request by mikecroft**

### CODE REVIEW

File: documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+[[stop-domains]]
+== `stop-all-domains`
+
+*Usage*::
+`asadmin> stop-all-domains`
+
+*Aim*::
+This command can be used to stop all domains detected. This command extends the
```
Comment (OndrejM) [inline documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc]: "...to stop all detected domains."
Isn't this better?

File: documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+|false
+|No
+
+|===
+
+=== Example
+
+[source]
+----
+asadmin> stop-all-domains
```
Comment (OndrejM) [inline documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc]: Why is this on 2 lines? I'd put it in a single line:
```
asadmin>  stop-all-domains --force=false

File: documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+|`--force`
+|Boolean
+|Specifies whether each domain is forcibly stopped immediately.
+|true
+|No
+
+|`--kill`
+|Boolean
+|Specifies whether each domain is forcibly stopped immediately.
```
Comment (OndrejM) [inline documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc]: The description is incorrectly copied from `--force`. It should say:

> Specifies whether the domain is killed by using functionality of the operating system to terminate the domain process

(copied from the stop-domain man page)

File: documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc (hunk 2 above)
Comment (mikecroft) [inline documentation/extended-documentation/asadmin-commands/asadmin-commands.adoc]: I was following the convention from the page I copied which had a much longer list. I agree this looks better here.

**Pull request merged**